components/global/product_switcher.tsx

```diff
@@ -73,11 +87,25 @@ const MenuItemTextContainer = styled.div`
     line-height: 20px;
 `;
 
-const LinkIcon = styled.i`
-    width: 14px;
-    height: 14px;
-    color: rgba(var(--center-channel-color-rgb), 0.56);
-`;
+const SwitcherNavEntry = (props: SwitcherNavEntryProps) => {
+    return (
+        <MenuItem
+            to={props.destination}
+        >
+
+            <StyledIcon glyph={props.icon}/>
```

Comment:
This still needs to be addressed. Ether revert to a raw `{props.icon}` or work with Boards team to make sure we don't break community.

Reply:
It will be better to provide a string of the desired product icon glyph from Compass Icons instead of an icon component to ensure consistency of usage.

For this PR we can revert to the raw `{props.icon}` to avoid issues when merging and can then ensure that we talk with the product teams about the change.

@nevyangelova, lets go with this change for now and we can update in the next PR
```suggestion
            {props.icon}
```

Reply:
@nevyangelova cc @deanwhillier @crspeller perhaps something along these lines for the time being?

```tsx
const SwitcherNavEntry = (props: SwitcherNavEntryProps) => {
    let icon: React.ReactNode;
    if (isValidElementType(props.icon) && typeof props.icon !== 'string') {
        const ProductIcon = props.icon;
        icon = <ProductIcon/>;
    } else if (React.isValidElement(props.icon)) {
        icon = props.icon;
    } else if (typeof props.icon === 'string') {
        icon = <StyledIcon glyph={props.icon as TIconGlyph}/>;
    }

    return (
        <MenuItem
            to={props.destination}
        >
            {icon}
            <MenuItemTextContainer>
                {props.text}
            </MenuItemTextContainer>
            {props.active &&
                <Icon
                    size={16}
                    glyph='check'
                />
            }
        </MenuItem>
    );
};
```